docs/luigi-core-api.md

```diff
@@ -172,6 +172,47 @@ Luigi.unload()
 
 -   **since**: 1.2.2
 
+#### readSettingsFromStorage
+
+Reads the user settings object from the localstorage. It is possible to choose a custom storage type by implementing the `settings.userSetting.readFromCustomStorage` function.
+
+##### Parameters
+
+-   `key` **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** used to identify the item in the storage.
+-   `property` **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** the object traversal path where `readFromCustomStorage` function is implemented. Optional and only needed if there is a custom implementation of `readFromCustomStorage` function in the settings section of the luigi config to read a custom storage.
+
+##### Examples
+
+```javascript
+Luigi.writeUserSettings('key');
+Luigi.writeUserSettings('key', 'settings.userSettings');
```

Comment:
```suggestion
Luigi.readFromCustomStorage('key');
Luigi.readFromCustomStorage('key', 'settings.userSettings');
```

Reply:
Done, it is named `writeSettingsToStorage`. Thx for finding, hope name is ok.